examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p><strong>Note</strong>: after looking at the each model's MAPE, one would notice that <code>NaiveSeasonal</code> is actually performing better on its own than ensembled with <code>NaiveDrift</code>. Checking the performance of the single models is in general a good practice before defining an ensemble. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>